Add default text in System Images when no image is present in Admin Panel

harsh-jindal opened the pull request:
<!-- 
(Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.)
-->

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes <!-- use `ember test` to run all the tests -->
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Fix Bug in System Images in Admin. Before a blank page appears if a new event topic was created without event subtopic as system images are only for subtopics. Now a text "No records to show" appears.

#### Changes proposed in this pull request:
![Screenshot 2019-04-25 at 2 45 53 AM](https://user-images.githubusercontent.com/31013104/56694505-77443680-6704-11e9-9b73-4b7c6f47994a.png)

<!-- Add the issue number that is fixed by this PR (In the form Fixes #123) -->

Fixes #2690

ritikamotwani commented on app/templates/admin/content/system-images/list.hbs:
```diff
@@ -1,12 +1,16 @@
+        <div class="ui hidden divider"></div>
+        <button class="ui button primary" {{action 'openModal' subTopic.placeholder}} id="changebutton">{{t 'Change'}}</button>
+      </div>
+    {{/each}}
```
Use each else syntax.
https://guides.emberjs.com/v1.13.0/templates/displaying-a-list-of-items/#toc_empty-lists

ritikamotwani commented on app/templates/admin/content/system-images/list.hbs:
```diff
@@ -1,12 +1,16 @@
+      </div>
+    {{/each}}
+  {{else}}
+    <div style="font-size:24px; padding:10px;">No system Images</div>
```
Check how helper text is added in rest of the places.

harsh-jindal commented:
@ritikamotwani Made the changes, please review.

ritikamotwani commented on app/templates/admin/content/system-images/list.hbs:
```diff
@@ -7,6 +7,10 @@
       <button class="ui button primary" {{action 'openModal' subTopic.placeholder}} id="changebutton">{{t 'Change'}}</button>
     </div>
+  {{else}}
+    <div class="ui basic segment">
```
No, this is not how helper text is added in the rest of the tables of the app.

harsh-jindal commented:
@CosmicCoder96 I have changed the description and the commit message. Will keep the above point in mind from next time onwards.
@ritikamotwani I have implemented this the same as in #2517. 
Please review

ritikamotwani commented:
@harsh-jindal , in the PR you've mentioned the helper text had to be like the other tables in that page itself.
For the rest of the tables "disabled header" class is used.
Check how it's done in the other places.

CosmicCoder96 commented:
@harsh-jindal Update this.

harsh-jindal commented:
@CosmicCoder96 @ritikamotwani Made the changes. Please review.
